public/stylesheets/css/login.css

```diff
@@ -0,0 +1,266 @@
+/* Import Lato Font */
+@import url("https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700");
```

Comment:
Might as well serve lato locally? To minimise http requests. Not sure what the License is like on it